airbyte-integrations/connectors/source-googleanalytics-singer/source_googleanalytics_singer/source.py

```diff
@@ -59,19 +61,26 @@ def _validate_custom_reports(self, custom_reports_data: List[dict]):
                 error_messages.append(error.message)
             raise Exception("An error occurred during custom_reports data validation: " + "; ".join(error_messages))
 
+    def read_catalog(self, catalog_path: str) -> ConfiguredAirbyteCatalog:
```

Comment:
this method is not called anywhere?

Reply:
It is called when performing read operation, I redefined this function from a base class to save and then filter out streams we want to use

Reply:
I got it

Reply:
@yevhenii-ldv for now it works as before